server/src/main/kotlin/org/javacs/kt/rename/Rename.kt

```diff
@@ -28,7 +28,7 @@ fun renameSymbol(file: CompiledFile, cursor: Int, sp: SourcePath, newName: Strin
     }
 }
 
-private fun findDeclaration(file: CompiledFile, cursor: Int): Pair<KtNamedDeclaration, Location>? {
+fun findDeclaration(file: CompiledFile, cursor: Int): Pair<KtNamedDeclaration, Location>? {
```

Comment:
If we use this function elsewhere, would it make sense to move it into a utility module? Or directly into `CompiledFile` as a method?